bokeh/models/annotations.py

```diff
@@ -879,9 +879,12 @@ class Title(TextAnnotation):
     The text value to render.
     """)
 
-    align = Enum(TextAlign, default='left', help="""
-    Location to align the title text.
+    vertical_align = Enum(VerticalAlign, default='bottom', help="""
+    Vertical aligment of the text.
+    """)
 
+    align = Enum(TextAlign, default='left', help="""
+    Horizontal aligment of the text.
     """)
```

Comment:
We might want to mark this for re-visting before release. I think it might be better to always just have one "align" which is agnostic to orientation. It is always relative *to the direction of the text*, whether horizontal or vertical. Then the setting makes sense even if you switch from horizontal to vertical (this way does not)

Reply:
This docstring is actually wrong. This `align` relates to the "major" dimension, so horizontal if panel is above or below, and vertical if left or right. `vertical_align` acts on the orthogonal dimension.